### DISCUSSION TITLE
Creacion de una base de datos para `Publicacion`/`Programa`?

### DISCUSSION THREAD

**Discussion by JUNIORRDSR** [Curcolor/COMMUNITY#17] [Ideas]
# Añadir la Tabla de 'Publicacion' a la base de datos 'Usuarios' o crear una base de datos para esta?
Viendo que la unica información que tenemos en el documento ERS sobre los programas es esto:

> 1. Presentación de información: El sistema debe presentar la información de manera clara y estructurada, permitiendo a los usuarios navegar fácilmente entre las opciones de programas de ayuda. Esto debe incluir:
> > - Tarjetas informativas: Cada programa debe mostrarse en un formato de tarjeta que incluya datos como el nombre del programa, una breve descripción, el tipo de ayuda, y un enlace para acceder a más información.
> > - Detalles adicionales: Al hacer clic en un programa, se debe desplegar información detallada, incluyendo:
> > > - Descripción completa del servicio de ayuda.
> > > - Requisitos o restricciones (si los hay).
> > > - Contacto directo o enlace a la plataforma que ofrece la ayuda.
> > >- Información sobre disponibilidad 

Que opinan.
> [!note]
> Adjunto imagen de como se vería `Publicacion` en una base de datos separada.
![Base de datos Programas-Publicacion](https://github.com/user-attachments/assets/0217dd14-fdb6-428d-a748-b64f0dc7daf9)
> Adjunto imagen de como se vería `Publicacion` en una base de datos junto con `Usuarios`.
![Base de datos Usuarios-Voluntarios-Programas](https://github.com/user-attachments/assets/74ee67e0-5549-4ffe-800c-5638d45a219b)

**Comment by Curcolor** [comment]
@JUNIORRDSR  Yo opino que que se cree mejor otra tabla llamada `Programa` o mejor ponerle el nombre de `Publicaciones_externas` y estas junto a la primera opción hagan una base de datos para simplificar la programación pues serán estas tablas que se conectarán con las Apis externas y con todo el _web scraping_.
